### DISCUSSION TITLE
在移动端的显示的情况下给操作按钮增加文字显示

### DISCUSSION THREAD

**Discussion by webrjb** [dotnetcore/BootstrapBlazor#3490] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

在PC端按钮可以显示操作按钮名称，比如新增，删除等，但是在移动端情况下，按钮就没有这些字样了，只有图标，希望大佬考虑增加按钮文字显示，这样对用户更友好，谢谢!
![搜狗截图20240509094243](https://github.com/dotnetcore/BootstrapBlazor/assets/87313546/be80110b-4067-4806-aa81-8de44f925ac6)

**Comment by bb-auto** [comment]
@webrjb Thank you for reporting. We will give triage later.

**Reply by ArgoZhang** [reply, marked as answer]
@webrjb 感谢提交建议

1. 移动端没有 `hover` 需要点一下才出 `Tooltip` 第二次点击才是功能，这个无个人无法接受
2. 加 `title` 属性，PC 端又丑

觉得这个回答满意，请点一下下面的 `Make as answer` 按钮
